modules/mono/csharp_script.cpp

```diff
@@ -304,6 +304,25 @@ void CSharpLanguage::get_reserved_words(List<String> *p_words) const {
 	}
 }
 
+bool CSharpLanguage::is_control_flow_keyword(String p_keyword) const {
+	return p_keyword == "break" ||
+		   p_keyword == "case" ||
+		   p_keyword == "catch" ||
+		   p_keyword == "continue" ||
+		   p_keyword == "default" ||
+		   p_keyword == "do" ||
+		   p_keyword == "if" ||
```

Comment:
I think the goal here was to make this in alphabetical order, but then `if` is out of place. Same with GDScript (but then there's also match/return out of order).